xarray/core/variable.py

```diff
@@ -706,8 +706,9 @@ def transpose(self, *dims):
         data = ops.transpose(self.data, axes)
         return type(self)(dims, data, self._attrs, self._encoding, fastpath=True)
 
-    def expand_dims(self, dims, shape=None):
-        """Return a new variable with expanded dimensions.
+    def set_dims(self, dims, shape=None):
```

Comment:
Can you keep around `Variable.expand_dims` as an alias for `set_dims` with a deprecation warning?

e.g., see the deprecation warning for `broadcast_arrays`: https://github.com/pydata/xarray/blob/93d6963315026f87841c7cf39cc39bb78f555345/xarray/core/alignment.py#L531